testing/test-setup/src/lib/chrome-path.setup.ts

```diff
@@ -2,11 +2,29 @@ import { getChromePath } from 'chrome-launcher';
 import * as process from 'node:process';
 import { beforeEach, vi } from 'vitest';
 
-beforeEach(() => {
+beforeEach(async () => {
   const customChromePath = process.env['CUSTOM_CHROME_PATH'];
 
   if (customChromePath == null) {
-    vi.stubEnv('CHROME_PATH', getChromePath());
+    try {
+      const path = getChromePath();
+      vi.stubEnv('CHROME_PATH', path);
+    } catch (error) {
+      if ((error as Error).message.includes('No Chrome installations found.')) {
+        const chromium = (await import('chromium' as string)) as Record<
+          'path',
+          string
+        >;
+        console.info(
+          `${
+            (error as Error).message
+          } Using chromium from node_modules instead: ${chromium.path}`,
+        );
+        vi.stubEnv('CHROME_PATH', chromium.path);
+      } else {
+        throw error;
+      }
```

Comment:
A runtime check for `Error` instance is safer and more convenient than repeating `(error as Error).message`.

```suggestion
      if (
        error instanceof Error &&
        error.message.includes('No Chrome installations found.')
      ) {
        const chromium = (await import('chromium' as string)) as Record<
          'path',
          string
        >;
        console.info(
          `${error.message} Using chromium from node_modules instead: ${chromium.path}`,
        );
        vi.stubEnv('CHROME_PATH', chromium.path);
      } else {
        throw error;
      }
```

Reply:
did not know that, THX!